### PR TITLE
Split grubby args and avoid need to quote

### PR DESCRIPTION
**Pull request by AloisMahdal**
The --foo=bar syntax can be confusing at times, so let's avoid it.

Fixes: #242

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py [hunk 1]
```diff
@@ -12,12 +12,12 @@ def add_boot_entry():
     kernel_dst_path, initram_dst_path = get_boot_file_paths()
     run([
         '/usr/sbin/grubby',
-        '--add-kernel={0}'.format(kernel_dst_path),
-        '--initrd={0}'.format(initram_dst_path),
-        '--title="RHEL Upgrade Initramfs"',
+        '--add-kernel', '{0}'.format(kernel_dst_path),
+        '--initrd', '{0}'.format(initram_dst_path),
+        '--title', 'RHEL Upgrade Initramfs',
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py]:
```suggestion
        '--title', '"RHEL Upgrade Initramfs"',
```
I would leave there the quotation marks, the same as you would use them in a terminal.
Neither stdlib _run_ or __call_ tell in description how the processing of the list works, so I'd better stay defensive.
CCing @shaded-enmity.

Comment (AloisMahdal) [reply repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py]: That's not defensive; you're just passing the text *with* quotes.

I'm not sure about `run` and `_call` but standard library passes the args right to the forked process without involving shell at all.

Comment (shaded-enmity) [reply repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py]: @bocekm @AloisMahdal The list of arguments is passed verbatim to `os.exec*` function.

Comment (shaded-enmity) [reply repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py]: I.e. 
```
os.exec('/bin/echo', ['/bin/echo', '"FOO"'])
"FOO" 
```

whereas

```
os.exec('/bin/echo', ['/bin/echo', 'FOO'])
FOO
```

So it depends on what Grubby expects.

**Comment by AloisMahdal** [comment]
Upon consulting in person, I've also added commit to make the unit test less confusing.

File: repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py (hunk 1 above)
Comment (AloisMahdal) [reply repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py]: Yep, that's what I expected.  (It's the safest way to call commands anyway...)

So quotes/spaces/any-old-junk does not really influence "safety".

**Comment by AloisMahdal** [comment]
Already tested with `leapp-repository-0.7.0-0.201906111421Z.1d63bb7.grubby_split_args.PR243.el7_6.noarch`.  That does not contain the later commit but the later commit is only relevant to unit tests, so that one will be tested by CI.

**Comment by AloisMahdal** [comment]
> ..so it will be tested by CI.

and tested it was, and fail it did.  I made a typo; force-pushing.. .:)

**Pull request merged**